### PR TITLE
Update Package.swift and README.md on `feature/swift 5.2` branch

### PR DESCRIPTION
**Pull request by alfavata**

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.1
+// swift-tools-version:5.2
```
Comment (fabianfett) [inline Package.swift]: if we increase the version here, every importing package must also be on swift 5.2 afaik. I'm not sure if we want to drop support for 5.1 yet?

**Review by fabianfett — COMMENTED** [review]
Thanks for updating the readme. I'm not sure though if the `Package.swift` change is what we want.

File: Package.swift (hunk 1 above)
Comment (alfavata) [reply Package.swift]: Fair enough! I'll change that back

Comment (alfavata) [reply Package.swift]: I think we still need the other changes in `Package.swift` when building with Xcode 11.4 though

**Pull request merged**